doc/index.md

```diff
@@ -116,6 +117,23 @@ align: center
 ---
 :::
 
+:::{tab-item} DuckDB
```

Comment:
It doesn't seem like DuckDB is used in this example?

Reply:
You're right :D I copied and pasted but forgot to add the duckdb.from_df part; added now.